vcf-dumper/vcf-dumper-lib/src/test/java/embl/ebi/variation/eva/vcfdump/regionutils/RegionFactoryTest.java

```diff
@@ -88,4 +106,29 @@ public void divideRegionInChunks() {
         regions = regionFactory.divideRegionInChunks("1", 3000, 2500);
         assertTrue(regions.size() == 0);
     }
+
+    @Test
+    public void executeQueryForGettingMinAndMaxCoordinatesWhenTheyAreNotIncludedInTheRegionFilter() throws IOException, IllegalOpenCGACredentialsException {
```

Comment:
Isn't this method actually testing 2 different cases?

1. Regions with specified limits (1:500-2499 and 21:1000-2000)
2. Regions with unspecified limits (22)